client/src/CommentBox.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import $ from 'jquery';
+import CommentForm from './AddCommentForm';
+import CommentList from './CommentList';
+
+class CommentBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [] };
+  }
+
+  componentDidMount = () => {
+    this.loadComments();
+    setInterval(this.loadComments, this.props.pollInterval);
+  }
+
+  loadComments = () => {
+    $.ajax({
+      url: this.props.url.concat('/', this.props.poemId),
+      dataType: 'json',
+      cache: false,
+      success: (data) => {
+        this.setState({ data });
+      },
+      error: (xhr, status, err) => {
+        console.error(this.props.url, status, err.toString());
+      },
+    });
+  }
+
+  handleCommentSubmit = (comment) => {
+    $.ajax({
+      url: this.props.url.concat('/', this.props.poemId),
+      dataType: 'json',
+      type: 'POST',
+      contentType: 'application/json',
+      data: JSON.stringify(comment),
+      success: (data) => {
+        this.setState({ data });
+      },
+      error: (xhr, status, err) => {
+        console.error(this.props.url, status, err.toString());
+      },
+    });
+  }
+
+  render = () => (
+    <div className="commentBox">
+      <h2>Comments</h2>
+      <CommentList data={this.state.data} />
+      <CommentForm onCommentSubmit={this.handleCommentSubmit} poemId={this.props.poemId} />
```

Comment:
👍 A nice benefit is that the client can't accidentally -- or purposefully -- be inconsistent by adding the comment in the context of one poem but setting the ID to that of a different poem when sending it to the server.